projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-grid-navigation.service.ts

```diff
@@ -67,6 +72,35 @@ export class IgxPivotGridNavigationService extends IgxGridNavigationService {
         }
     }
 
+    public override handleAlt(key: string, event: KeyboardEvent): void {
+        event.preventDefault();
+
+        let row = this.grid.gridAPI.get_row_by_index(this.activeNode.row);
+        let expansionRowKey = PivotUtil.getRecordKey(row.data, row.data.dimensions[0])
+        let isExpanded = this.grid.expansionStates.has(expansionRowKey) ? this.grid.expansionStates.get(expansionRowKey) : true;
+
+        if (!isExpanded && ROW_EXPAND_KEYS.has(key)) {
+            if (row.key === undefined) {
+                // TODO use expanded row.expanded = !row.expanded;
+                (row as any).toggle();
+            } else {
+                const newExpansionState = new Map<any, boolean>(this.grid.expansionStates);
+                newExpansionState.set(expansionRowKey, true);
+                this.grid.expansionStates = newExpansionState;
```

Comment:
This is fine, but it probably won't trigger any events, like the `rowToggle` event. Would be better to use the existing api:
`this.grid.gridAPI.set_row_expansion_state`